core/src/main/java/org/apache/iceberg/avro/SupportsCustomRecords.java

```diff
@@ -20,7 +20,7 @@
 
 import java.util.Map;
 
-/** An interface for Avro DatumReaders to support custom record classes. */
+/** An interface for Avro DatumReaders to support custom record classes by name. */
```

Comment:
This is used to distinguish between Iceberg custom types that used field IDs and `StructLike` and Avro's, which worked by renaming Avro records to class names that would be dynamically loaded.